package.json

```diff
@@ -23,33 +23,33 @@
     ]
   },
   "dependencies": {
-    "@hapi/accept": "^5.0.1",
-    "@hapi/ammo": "^5.0.1",
-    "@hapi/boom": "^9.1.0",
-    "@hapi/bounce": "^2.0.0",
-    "@hapi/call": "^8.0.0",
-    "@hapi/catbox": "^11.1.1",
-    "@hapi/catbox-memory": "^5.0.0",
-    "@hapi/heavy": "^7.0.1",
-    "@hapi/hoek": "^9.0.4",
-    "@hapi/mimos": "^6.0.0",
-    "@hapi/podium": "^4.1.1",
-    "@hapi/shot": "6.0.0-beta.1",
-    "@hapi/somever": "^3.0.0",
-    "@hapi/statehood": "^7.0.3",
-    "@hapi/subtext": "^7.0.3",
-    "@hapi/teamwork": "^5.1.0",
-    "@hapi/topo": "^5.0.0",
-    "@hapi/validate": "^1.1.1"
+    "@hapi/accept": "^6.0.0",
+    "@hapi/ammo": "^6.0.0",
+    "@hapi/boom": "^10.0.0",
+    "@hapi/bounce": "^3.0.0",
+    "@hapi/call": "^9.0.0",
+    "@hapi/catbox": "^12.0.0",
+    "@hapi/catbox-memory": "^6.0.0",
+    "@hapi/heavy": "^8.0.0",
+    "@hapi/hoek": "^10.0.0",
+    "@hapi/mimos": "^7.0.0",
+    "@hapi/podium": "^5.0.0",
+    "@hapi/shot": "^6.0.0",
+    "@hapi/somever": "^4.0.0",
+    "@hapi/statehood": "^8.0.0",
+    "@hapi/subtext": "^8.0.0",
+    "@hapi/teamwork": "^6.0.0",
+    "@hapi/topo": "^6.0.0",
+    "@hapi/validate": "^2.0.0"
   },
   "devDependencies": {
-    "@hapi/code": "9.0.0-beta.0",
-    "@hapi/eslint-plugin": "^5.0.0",
+    "@hapi/code": "^9.0.0",
+    "@hapi/eslint-plugin": "^6.0.0",
```

Comment:
One thing though, I remember we set this dep to `'*'` with @Marsup in some plugins because we were having issues without it. Perhaps the underlying issue that was causing the behaviour has been fixed and I'm not up to date but I figured I might just voice it.

Reply:
In theory that makes perfect sense to me, but in practice this has been causing build failures.  Here's a failed run for hapi v20 that can be fixed by specifying the eslint-plugin version: https://github.com/hapijs/hapi/runs/6866679357?check_suite_focus=true.  These started failing once the next major of eslint-plugin was published.